server/views/pages/activities/create-an-activity/pay-rate-type.njk

```diff
@@ -5,7 +5,7 @@
 
 {% set pageTitle = applicationName + " - Create an activity - Pay Rate option" %}
 {% set pageId = 'create-activity-pay-rate-type-page' %}
-{% set backLinkHref = "risk-level" %}
+{% set backLinkHref = "/activities/schedule/activities/" + session.createJourney.activityId if session.req.query.fromEditActivity else "risk-level" %}
 
```

Comment:
Should the `session.req.query.fromEditActivity` part be prefixed `session`? Or just `req.query.fromEditActivity`?